add_annotations_to_table_helper.py

```diff
@@ -104,7 +109,9 @@ def transcript_for_alt(transcripts, alt):
             for alt in entry['ALT'].split(','):
                 vep_annotations = vep[entry['CHROM']][entry['POS']][entry['REF']][alt]
                 if vep_annotations is not None and field in vep_annotations:
-                    field_annotations.append(vep_annotations[field])
+                    annotation = vep_annotations[field]
+                    decoded_annotation = re.sub(r'%[0-9|A-Z][0-9|A-Z]', decode_hex, annotation)
```

Comment:
If this is decoding hex, should it it only match on 0-9 and A-F?

Reply:
Fixed.